src/modules/util/node-fetch-har.d.ts

```diff
@@ -1 +1,9 @@
-declare module 'node-fetch-har';
+declare module 'node-fetch-har' {
```

Comment:
> Could not find a declaration file for module 'node-fetch-har'. '/home/kcd/projects/platformer/cli/node_modules/node-fetch-har/index.js' implicitly has an 'any' type.
  Try `npm install @types/node-fetch-har` if it exists or add a new declaration (.d.ts) file containing `declare module 'node-fetch-har';`

@binura-g any idea on how to fix this? vscode correctly picks up the types. `yarn test` doesn't.

Reply:
@ChathurangaKCD just set it to ignore the line

```ts
// @ts-ignore
import { createHarLog, withHar } from 'node-fetch-har';
```

Reply:
Fixed this